bytecode/concrete.py

```diff
@@ -205,7 +204,7 @@ def from_code(code, *, extended_arg=False):
         bytecode = ConcreteBytecode()
         bytecode.name = code.co_name
         bytecode.filename = code.co_filename
-        bytecode.flags = code.co_flags
+        bytecode.flags = _bytecode.Flags(code.co_flags)
```

Comment:
Do you really need to explicitly convert to Flags? The setter does it for you, no? If not, please revert this change.